### PR TITLE
To sample each file to 64*64

### PR DESCRIPTION
**Pull request by adityavashistee**

### CODE REVIEW

File: Sampler/BDE-Sampler.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+from skimage import io
+from skimage.transform import resize
+from scipy import misc
+from PIL import Image
+import glob
+#CURE-TSR/Real_Train/*/*.bmp
+i=1
+for filename in glob.glob('CURE-TSR/Real_Train/*/*.bmp'): #read all bmp files from all the folders in Real_Train folder
+    im= io.imread(filename)
+    g=resize(im, (64, 64),mode='reflect') #resize the image to 64*64 size
```
Comment (rachit-ranjan16) [inline Sampler/BDE-Sampler.py]: Extract Resize to Appconfig

File: Sampler/BDE-Sampler.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# from skimage import data
+# from matplotlib import pyplot as plt
+from skimage import io
+from skimage.transform import resize
+from scipy import misc
+from PIL import Image
+import glob
+#CURE-TSR/Real_Train/*/*.bmp
+i=1
+for filename in glob.glob('CURE-TSR/Real_Train/*/*.bmp'): #read all bmp files from all the folders in Real_Train folder
```
Comment (rachit-ranjan16) [inline Sampler/BDE-Sampler.py]: Extract destination folder to appconfig

File: Sampler/BDE-Sampler.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# resize image to 64*64 to standardize the input for the training data
+# from skimage import data
+# from matplotlib import pyplot as plt
+from skimage import io
+from skimage.transform import resize
+from scipy import misc
+from PIL import Image
+import glob
+#CURE-TSR/Real_Train/*/*.bmp
+i=1
```
Comment (rachit-ranjan16) [inline Sampler/BDE-Sampler.py]: Refer 
https://github.com/rachit-ranjan16/foodie/config 
for appConfig Extraction

**Comment by adityavashistee** [comment]
these changes will be part of next increment

**Pull request merged**